roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -173,41 +173,89 @@ keyring = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hos
 log file = /var/log/ceph/{{ cluster }}-rgw-{{ hostvars[host]['ansible_hostname'] }}.log
 {% if hostvars[host]['radosgw_address_block'] is defined and hostvars[host]['radosgw_address_block'] != 'subnet' %}
     {% if ip_version == 'ipv4' %}
-rgw frontends = civetweb port={{ hostvars[host]['ansible_all_' + ip_version + '_addresses'] | ipaddr(radosgw_address_block) | first }}:{{ radosgw_civetweb_port }} {{ radosgw_civetweb_options }}
+	    {% if radosgw_frontend == 'civetweb' %}
+rgw frontends = {{ radosgw_frontend }} port={{ hostvars[host]['ansible_all_' + ip_version + '_addresses'] | ipaddr(radosgw_address_block) | first }}:{{ radosgw_frontend_port }} {{ radosgw_frontend_options }}
+	    {% elif radosgw_frontend == 'beast' %}
+rgw frontends = {{ radosgw_frontend }} port={{ radosgw_frontend_port }} {{ radosgw_frontend_options }}
+	    {% endif %}
```

Comment:
the beast case should look exactly the same as the civetweb one, except `port=` should be `endpoint=`